README.md

```diff
@@ -1,3 +1,5 @@
+![gh-actions/site](https://github.com/monero-project/monero-site/workflows/gh-actions/site/badge.svg)
```

Comment:
```suggestion
![gh-actions/site](https://github.com/monero-project/monero-site/workflows/gh-actions/site/badge.svg?branch=master)
```

Reply:
If a branch is not specified it uses the default branch selected on GitHub, which in our case is master, that's why i didn't specify it before. Changing it anyway.

Reply:
ah true, yea guess then it isn’t necessary